server/core/src/main/java/dev/slimevr/tracking/trackers/udp/UDPDevice.kt

```diff
@@ -53,6 +53,9 @@ class UDPDevice(
 	var timedOut = false
 	override val trackers = ConcurrentHashMap<Int, Tracker>()
 
+	@JvmField
```

Comment:
I honestly prefer not having these if we are adding a new field, it was mainly for when we were migrating old stuff to Kotlin
```suggestion
```